4-Express/server/mychecklist_server.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express')
```

Comment:
the name of the file is a bit weird. how about `todos_service` or something similar? 
you could also split this file into `index.js` that will configure and start the server and `todos_[controller/service].js` to manage the routes